src/main/java/com/javalearn/circle/entity/Circle.java

```diff
@@ -0,0 +1,41 @@
+package com.javalearn.circle.entity;
+
+public class Circle {
+    private Point centre;
```

Comment:
better to make class immutable by default i.e. add final here and remove setters